core/src/main/java/jenkins/security/ResourceDomainConfiguration.java

```diff
@@ -134,7 +134,7 @@ private FormValidation checkUrl(String resourceRootUrlString, boolean allowOnlin
 
         StaplerRequest currentRequest = Stapler.getCurrentRequest();
         if (currentRequest != null) {
-            String currentRequestHost = currentRequest.getHeader("Host");
+            String currentRequestHost = String.format("%s", currentRequest.getServerName());
```

Comment:
This could just be written as ```String currentRequestHost = currentRequest.getServerName();``` to avoid the additional call to ```String.format```. It will always return a ```String```.